src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -35,6 +35,7 @@
  * throws a violation. A continuation line is when the description starts/spans past the line with
  * the tag. Default indentation required is at least 4, but this can be changed with the help of
  * properties below.
+ * Note: This check does not validate the indentation of lines inside {@code pre} tags.
```

Comment:
this update , as separate PR is not not valid.

https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java has no logic on PRE.

delivery of this update is separate PR/commit will simplify sizing of PRs but will make it more confusing in long run when people will question, WHY this Check is doing PRE in this way? the only way to get answers is to open history and be linked to issues and PRs to get answer.
That is why we keep all together, to make clear by history. Nobody should remember why something is done in certain way, system should do this.

Please move this update to PR where you add PRE.